lib.Makefile

```diff
@@ -257,23 +262,13 @@ GOARCH_FLAGS :=-e GOARCH=$(ARCH)
 REPO_ROOT := $(shell git rev-parse --show-toplevel)
 CERTS_PATH := $(REPO_ROOT)/hack/test/certs
 
-# Set the platform correctly for building docker images so that
-# cross-builds get the correct architecture set in the produced images.
-ifeq ($(ARCH),arm64)
-TARGET_PLATFORM=--platform=linux/arm64/v8
-endif
-ifeq ($(ARCH),ppc64le)
-TARGET_PLATFORM=--platform=linux/ppc64le
-endif
-ifeq ($(ARCH),s390x)
-TARGET_PLATFORM=--platform=linux/s390x
-endif
+QEMU_IMAGE ?= jha48/qemu-user-static:latest
```

Comment:
- [ ] Update when the image location is finalized.

Reply:
The custom `calico/qemu-user-static` serves as a holder for all qemu-{amd64,arm64,ppc64le,s390x}-static binaries we need when cross-compiling. We used to copy from `calico/go-build` which contains the same files but it is big and takes time to extract and copy qemu files.